frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -1,101 +1,101 @@
-  </ListHeader>
-);
 
-const VMRow = ({ obj: vm }: React.ComponentProps<typeof ResourceRow>) => {
+const tableColumnClasses = [
+  classNames('col-lg-4', 'col-md-4', 'col-sm-6', 'col-xs-6'),
+  classNames('col-lg-4', 'col-md-4', 'hidden-sm', 'hidden-xs'),
+  classNames('col-lg-4', 'col-md-4', 'col-sm-6', 'col-xs-6'),
+  Kebab.columnClass,
+];
+
+const VMHeader = () => [
+  {
+    title: 'Name',
+    sortField: 'metadata.name',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[0] },
+  },
+  {
+    title: 'Namespace',
+    sortField: 'metadata.namespace',
+    transforms: [sortable],
+    props: { className: tableColumnClasses[1] },
+  },
+  // {
+  //   title: 'Status',
+  //   props: { className: tableColumnClasses[2] },
+  // },
+  // {
+  //   title: '',
+  //   props: { className: Kebab.columnClass, props: { className: tableColumnClasses[3] } },
+  // },
+];
+
+const VMRow: React.FC<VMRowProps> = ({ obj: vm, index, key, style }) => {
   const name = getName(vm);
   const namespace = getNamespace(vm);
 
-  /*
-  TODO(mlibra) To keep recent PR minimal, relations to other objects will be migrated as a next step.
-               Keeping the code here for reference.
-  const migrationResources = getResource(VirtualMachineInstanceMigrationModel, {namespace});
-  const resourceMap = {
-    pods: {
-      resource: getResource(PodModel, { namespace }),
-    },
-    migrations: {
-      resource: migrationResources,
-    },
-  };
-  */
-
   return (
-    <ResourceRow obj={vm}>
-      <div className={mainRowSize}>
+    <TableRow id={getUid(vm)} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses[0]}>
         <ResourceLink kind={VirtualMachineModel.kind} name={name} namespace={namespace} />
-      </div>
-      <div className={otherRowSize}>
+      </TableData>
+      <TableData className={tableColumnClasses[1]}>
         <ResourceLink kind={NamespaceModel.kind} name={namespace} title={namespace} />
-      </div>
-      {/* TODO(mlibra): migrate VM status in a follow-up
-    <div className={mainRowSize}>
-        <WithResources resourceMap={resourceMap} loaderComponent={() => DASHES}>
-          <VmStatus vm={vm}/>
-        </WithResources>
-    </div>
-    */}
-      {/* TODO(mlibra): migrate actions in a follow-up
-      <div className="dropdown-kebab-pf">
-      <ResourceKebab actions={menuActions} kind={VirtualMachineModel.kind} resource={vm} resources={[
-        getResource(VirtualMachineInstanceModel, { name, namespace, isList: false }),
-        getResource(VirtualMachineInstanceMigrationModel, {namespace}),
-        getResource(PodModel, {namespace}),
-      ]} />
-    </div>
-    */}
-    </ResourceRow>
+      </TableData>
+      {/* TODO(mlibra): migrate VM status in a follow-up */}
+      {/* TODO(mlibra): migrate actions in a follow-up */}
+    </TableRow>
   );
 };
 
-const VMList: React.FC = (props) => <List {...props} Header={VMHeader} Row={VMRow} />;
+type VMRowProps = {
```

Comment:
AFAIK types are usually put at the end of file in `public` folder. We should also follow that convention